### ISSUE TITLE
iOS login not working after app update

### ISSUE THREAD

**Issue by jklippel** [jitsi/jitsi-meet#11073]
### Description:

After updating the iOS app to 22.0.0 it is no longer possible to log in / start a meeting which requires a password.

### Steps to reproduce:

1. Open iOS jitsi app
2. Try to join a user/password protected room
3. experience bug

### Expected behavior:

A username and password prompt should appear.

### Actual behavior:

An endlessly  rotating circle is shown.

### Server information:

- Jitsi Meet version: latest docker images
- Operating System: Linux

### Client information:

- Browser / app version: 22.0.0
- Operating System: iOS

### Additional information:

It is still possible to join the meeting once the room is opened via an older app version or via the browser.

**Comment by jklippel** [comment]
Jitsi server version: stable-6865

**Comment by saghul** [comment]
Ouch. We'll take a look, thanks for the report. /cc @Calinteodor

**Comment by Calinteodor** [comment]
Thank you for letting us know! I don’t seem to reproduce this issue. Can you try to delete and reinstall the app? Maybe something went wrong when updating. I tried joining a password secured room and a prompt appears and a password is requested, at least this happens on my end.

https://user-images.githubusercontent.com/26413496/156718329-3278676a-e69c-4d4e-83c2-5965c819181b.MOV

**Comment by saghul** [comment]
Same here. I checked with a fresh Docker setup just to be sure, but cannot repro.

**Comment by jklippel** [comment]
Re-installing jitsi-meet does not help.

It seems to be requesting stuff on the webserver side, but it never gets passed that:

```
web_1      | **.**.**.** - - [04/Mar/2022:08:20:02 +0000] "POST /http-bind?room=test HTTP/2.0" 200 304 "-" "jitsi-meet/168 CFNetwork/1329 Darwin/21.3.0"
web_1      | **.**.**.** - - [04/Mar/2022:08:20:05 +0000] "POST /http-bind?room=test HTTP/2.0" 200 304 "-" "jitsi-meet/168 CFNetwork/1329 Darwin/21.3.0"
```

**Comment by jklippel** [comment]
Oh btw. I restarted the server side as well. I got a second docker-based installation which exposes the same behaviour. Is there any internet-accessible password-protected instance (It is not the room but the whole instance that requires a password) I can try? If I would get to a credential prompt I would know that the problem is in my server.

**Comment by saghul** [comment]
Ah wait. You mean you are using Prosody authentication?

**Comment by DmDS** [comment]
I confirm!  An endlessly rotating circle is shown.  secure domain.

**Comment by saghul** [comment]
Don't you get the prompt? I just tested and I do. I do need to tap on OK twice for some reason though.

**Comment by saghul** [comment]
Ah no, it's actually the dialog not vanishing.

**Comment by saghul** [comment]
https://user-images.githubusercontent.com/317464/156733475-0954fe4f-501b-472c-b236-531b3f7f0859.MOV

**Comment by DmDS** [comment]
https://user-images.githubusercontent.com/1803032/156733864-606d3b05-3371-4b62-9b2e-0d454feeba12.MOV

**Comment by saghul** [comment]
Ah you are using the "auth with guests" flow. Let me try that.

**Comment by saghul** [comment]
Got it, I can repro now. Thanks for the help @DmDS and @jklippel, we'll fix this stat.

**Comment by saghul** [comment]
Fixed in https://github.com/jitsi/jitsi-meet/pull/11074 -- .1 release coming up.

**Comment by saghul** [comment]
I guess this is a good opportunity to plug our public iOS beta: https://testflight.apple.com/join/isy6ja7S :-) Hopefully we catch bugs before they end up in everyone's hands with some extra help!

**Comment by jklippel** [comment]
Joined the public beta and what can I say: You fixed it! Thank you so much!

Achievement unlocked: Fixed a bug within 24 hours after reporting!

Awesome!

**Comment by gcoretech** [comment]
Have any fixes been reverted? I've updated to jitsi meet 23.0.0 (build 1682342829) on ios and cannot start a meeting with username and password on my on-premise installation. Entering a wrong password will tell you so but nothing happens after entering the correct password. Hitting return or the OK button several times does nothing. It used to work with the latest 22.x app. I checked on another ios device with 22.x, it worked and after upgrading to 23 it stopped working. The password dialogue will not vanish. It is possible to attend an existing meeting though, the Jitsi ios app can join. In Safari (not using the Jitsi app) everything works normally. Server is running the latest 2.0.8319.
